pkgs/development/python-modules/rdkit/default.nix

```diff
@@ -0,0 +1,133 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, fetchzip
+, cmake
+, boost
+, catch
+, inchi
+, cairo
+, eigen
+, python
+, rapidjson
+, maeparser
+, coordgenlibs
+, numpy
+, stdenv
+, git
+}:
+buildPythonPackage rec {
+  version = "2020.09.3";
+  pname = "rdkit";
+
+  srcs = [
+    (fetchFromGitHub {
+      owner = pname;
+      repo = pname;
+      rev = "Release_2020_09_3";
+      sha256 = "1k1wvzcd1yfx4nhz3iq2aaadzdk5w6sfcb4imhvm7pkbzij0nicx";
+    })
+    (fetchzip {
+      url = "http://sourceforge.net/projects/avalontoolkit/files/AvalonToolkit_1.2/AvalonToolkit_1.2.0.source.tar";
+      sha256 = "1098k7p0bk1qhzxfx9ki1kg3ga9wbaj6zl0sbsiqdf59229ak060";
+    })
+    (fetchFromGitHub {
+      owner = "greglandrum";
+      repo = "yaehmop";
+      rev = "1b13b52e2738a77715b1bad876e3b4e93f2b5269";
+      sha256 = "1jp7wz8win4mgwxkaz2gnrgsaaqgln04n2lwgfr96isdv1klf62d";
+    })
+
+    (fetchFromGitHub
+      {
+        owner = "mittinatten";
+        repo = "freesasa";
+        rev = "2.0.3";
+        sha256 = "0x686zm9fpyg5647fdgxnxgbwav99nc6ymh4bmkr2063yyda4kzc";
+      })
+  ];
+
+  unpackPhase = ''
+    mkdir -p source/External/AvalonTools/avalon source/External/YAeHMOP/yaehmop source/External/FreeSASA/freesasa
+    cp -r ${builtins.elemAt srcs 0}/* source
```

Comment:
I don't really like this indexing by number. Wouldn't be an assigned variable easier to follow?